scipy/stats/_distribution_infrastructure.py

```diff
@@ -5219,3 +3582,288 @@ def __rtruediv__(self, other):
 
     def __neg__(self):
         return self * -1
+
+
+class Mixture(_ProbabilityDistribution):
+    r""" Class that represents a mixture distribution.
+
+    A mixture distribution is the distribution of a random variable
+    defined in the following way: first, a random variable is selected
+    from `components` according to the probabilities given by `weights`, then
+    the random variable is realized[1]_.
+
+    Parameters
+    ----------
+    components : sequence of `ContinuousDistribution`
+        The underlying instances of `ContinuousDistribution`.
```

Comment:
> It might be clearer to point out that `components` are the underlying random variables *to draw from*.

OK, I can add words.

As for the second part: If I felt free to to refer to instances of `ContinuousDistribution` as "random variables", I would have. I used this awkward wording to avoid trouble.